pkg/webui/console/containers/users-table/index.js

```diff
@@ -39,126 +52,254 @@ import {
+
+    return <Status status={indicator} label={label} pulse={false} />
+  },
+}
+
+const UsersTable = props => {
+  const { pageSize, currentUserId, maySendInvites, handleDeleteInvitation } = props
+  const dispatch = useDispatch()
+
+  const [tab, setTab] = React.useState(USERS_TAB)
+  const isInvitationsTab = tab === INVITATIONS_TAB
+
+  const onDeleteInvite = React.useCallback(
+    async email => {
+      try {
+        await handleDeleteInvitation(email)
+        toast({
+          message: m.revokeSuccess,
+          type: toast.types.SUCCESS,
+        })
+        setTab(INVITATIONS_TAB)
+        dispatch(getUserInvitations())
+      } catch {
+        toast({
+          message: m.revokeError,
+          type: toast.types.ERROR,
+        })
+      }
+    },
+    [handleDeleteInvitation, dispatch],
+  )
+
+  const headers = React.useMemo(() => {
+    const baseHeaders = []
+
+    if (tab === INVITATIONS_TAB) {
+      baseHeaders.push(
+        {
+          name: 'email',
+          displayName: sharedMessages.email,
+          width: 28,
         },
-      },
-      {
-        name: 'name',
-        displayName: sharedMessages.name,
-        width: 22,
-        sortable: true,
-      },
-      {
-        name: 'primary_email_address',
-        displayName: sharedMessages.email,
-        width: 28,
-        sortable: true,
-      },
-      {
-        name: 'state',
-        displayName: sharedMessages.state,
-        width: 15,
-        sortable: true,
-        render: state => {
-          let indicator = 'unknown'
-          let label = sharedMessages.notSet
-          switch (state) {
-            case 'STATE_APPROVED':
-              indicator = 'good'
-              label = sharedMessages.stateApproved
-              break
-            case 'STATE_REQUESTED':
-              indicator = 'mediocre'
-              label = sharedMessages.stateRequested
-              break
-            case 'STATE_REJECTED':
-              indicator = 'bad'
-              label = sharedMessages.stateRejected
-              break
-            case 'STATE_FLAGGED':
-              indicator = 'bad'
-              label = sharedMessages.stateFlagged
-              break
-            case 'STATE_SUSPENDED':
-              indicator = 'bad'
-              label = sharedMessages.stateSuspended
-              break
-          }
-
-          return <Status status={indicator} label={label} pulse={false} />
+        {
+          name: 'created_at',
+          displayName: m.sentAt,
+          width: 15,
+          render: date => <DateTime.Relative value={date} />,
         },
-      },
-      {
-        name: 'admin',
-        displayName: sharedMessages.admin,
-        width: 7,
-        render: isAdmin => {
-          if (isAdmin) {
-            return <Icon className={style.icon} icon="check" />
-          }
-
-          return null
+        ...state,
+        {
+          name: 'actions',
+          displayName: sharedMessages.actions,
+          width: 50,
```

Comment:
Note that these column widths are percentage values. This means that the sum of all column widths always needs to equal `100`. Please make sure that this is always the case in all configurations.